container_api/src/container.rs

```diff
@@ -178,13 +180,23 @@ fn create_context(_: &String, path: &String) -> Result<Context, HolochainError>
     let eav_path = format!("{}/eav", path);
     create_path_if_not_exists(&cas_path)?;
     create_path_if_not_exists(&eav_path)?;
+
+    let res = P2pNetwork::new(
+        Box::new(|_r| Ok(())),
+        &json!({
+            "backend": "mock"
```

Comment:
I assume this is temporary, as the real networking code isn't functional. Also, again, this should ultimately be a runtime config choice so that the end user can choose their networking solution.

Reply:
This blob is duplicated quite a bit in this PR. It may make sense to add a `with_mock()` function on `P2pNetwork` that can be used for unit tests / etc.

Reply:
I agree with both of these sentiments.  It is totally temporary because 1) we also don't have a lot of clarity on how to structure our tests in wake of all the CI clean, but more importantly in this particular case `container_api::create_context()` is also building file based storage, which really should also depend on the configuration file.   So I have added [a section to the tree](https://realtimeboard.com/app/board/o9J_kyiXmFs=/?moveToWidget=3074457346312589999) to account for this.